test/EIP712utils.ts

```diff
@@ -0,0 +1,176 @@
+import {
```

Comment:
I think [this comment](https://github.com/rsksmart/rif-relay/pull/232#discussion_r1088062741) also applies here

Reply:
Actually there are some utilities here that I still use. Although I;ve managed to reduce the file a lot. Please have a look.